README.md

```diff
@@ -32,7 +32,8 @@ An AI system can behave unfairly for a variety of reasons. In Fairlearn, we defi
 We follow the approach known as **group fairness**, which asks: _Which groups of individuals are at risk for experiencing harms?_ The relevant groups need to be specified by the data scientist and are application specific.
 
 Group fairness is formalized by a set of constraints, which require that some aspect (or aspects) of the AI system's behavior be comparable across the groups. The Fairlearn package enables assessment and mitigation of unfairness under several common definitions.
-To learn more about our definitions of fairness, please visit our [terminology page](./TERMINOLOGY.md#fairness-of-ai-systems).
+To learn more about our definitions of fairness, please visit our
+[terminology page](https://fairlearn.github.io/user_guide/fairness_in_machine_learning.html#fairness-of-ai-systems).
```

Comment:
It appears that the word 'terminology' doesn't actually appear on the linked page.... so I wouldn't call it a 'terminology page.'